api/test/integration/tavern_utils.py

```diff
@@ -88,3 +88,49 @@ def test_validate_upgrade_result(response, upgraded):
 
 def test_validate_update_latest_version(response):
     assert response.json().get('code', None) == 1749 or response.json().get('code', None) == 1718
+
+
+def test_count_elements(response, n_expected_items):
+    """
+    :param response: Request response
+    :param n_expected_items: Expected number of elements in affeted_items
+    """
+    assert len(response.json()['data']['affected_items']) == n_expected_items
+
+
+def test_expected_value(response, key, expected_values):
+    """
+    :param response: Request response
+    :param key: Key whose value to compare.
+    :param expected_values: Values to be found inside response.
+    """
+    expected_values = set(expected_values.split(',')) if not isinstance(expected_values, list) else set(expected_values)
+
+    for item in response.json()['data']['affected_items']:
+        response_set = set(item[key])
+        assert bool(expected_values.intersection(response_set)), \
+            f'Expected values {expected_values} not found in {item[key]}'
+
+
+def test_mitre_sort(response, attack_id, order):
```

Comment:
Modify test_sort_response to work with a single sort key or a list of keys, also with ascending or descending order. That way we keep it generic and available for all tests.

Reply:
This can't be achieved because the behaviour of sort parameter in /mitre endpoints is different to the behaviour in other endpoints right now. E. g:

If we have this items:
`1, 2, 3, 4, 5, 6, 7, 8, 9, 10`

When specifiying `sort: -<field>`, `limit: 3` in **/mitre**, it will return items depending on the sort field, but this items won't be sortered:
`8, 9, 10`

While in other endpoints (for instance /rules/requirement/{requirement}), it should be sorting the response:
`10, 9, 8`

In both cases it should be returning the last 3 elements, but not in the same order, so the current test_mitre_sort function can't be the same for both of them in my opinion.